### ISSUE TITLE
Draft Updated DOMO Dashboard

### ISSUE THREAD

**Issue by ATMiddleton** [department-of-veterans-affairs/va.gov-team#75604]
## Issue Description
_As a DOMO user, I need a dashboard that reflects the status of incoming Salesforce and AskVA cases, so I can clearly determine the volume and status of all cases including open, closed, backlogged cases for Tier II, Tier III, Tier III EVSS, and AskVA cases.

---
## Tasks
- [x] Identify case categories for tracking for Tier II, Tier III, Tier III EVSS, and AskVA 
- [x] Draft a grid displaying the total number of daily incoming cases 
- [x] Draft a grid displaying the number of cases by date frequency 
- [x] Draft a grid displaying the number of currently open cases 
- [x] Draft a grid displaying the number of currently closed cases
- [x] Draft a grid displaying the number of open backlog cases
- [x] Draft a grid displaying the number of closed backlog cases
- [x] Draft a grid displaying the number of routed cases 
- [x] Draft a grid displaying the number of cases routed to T3 EVSS 
- [x] Draft a grid displaying the number of cases routed to T3 
- [x] Draft a grid displaying the number of cases routed to T2 cases
- [x] Draft a grid displaying Total AskVA cases 
- [x] Draft a grid displaying total AskVA daily incoming cases 
- [x] Draft a grid displaying the number of cases by date frequency 
- [x] Determine that AskVA cases can be exported from Microsoft Dynamics to a csv file
- [x] Determine AskVA data reports can be automated for export to csv file 
- [x] Draft a grid displaying the number of currently open AskVA cases 
- [x] Draft a grid displaying the number of currently closed AskVA cases
- [x] Draft a grid displaying the number of open backlog AskVA cases
- [x] Draft a grid displaying the number of closed backlog AskVA cases
- [x] Draft a grid displaying the number of routed AskVA cases 
- [x]  Draft a grid displaying the number of cases routed from AskVA  to T3 including EVSS 

## Acceptance Criteria

- [x] **Scenario 1: case categories for tracking**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see the case categories for tracking, including Tier II, Tier III, Tier III EVSS, and AskVA

- [x] **Scenario 2: Display total number of daily incoming cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the total number of daily incoming cases

- [x] **Scenario 3: Display number of cases by date frequency**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of cases by date frequency

- [x] **Scenario 4: Display number of currently open cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of currently open cases

- [x] **Scenario 5: Display number of currently closed cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of currently closed cases

- [x] **Scenario 6: Display number of open backlog cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of open backlog cases

- [x] **Scenario 7: Display number of closed backlog cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of closed backlog cases

- [x] **Scenario 8: Display number of routed cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of routed cases

- [x] **Scenario 9: Display number of cases routed to T3 EVSS**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of cases routed to T3 EVSS

- [x] **Scenario 10: Display number of cases routed to T3**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of cases routed to T3

- [x] **Scenario 11: Display number of cases routed to T2 cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of cases routed to T2 cases

- [x] **Scenario 12: Display total AskVA cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the total number of AskVA cases

- [x] **Scenario 13: Display total AskVA daily incoming cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the total number of daily incoming AskVA cases

- [x] **Scenario 14: Display number of AskVA cases by date frequency**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of AskVA cases by date frequency

- [x] **Scenario 15: Determine AskVA cases can be exported to a csv file**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should be able to export AskVA cases to a csv file

- [x] **Scenario 16: Determine AskVA data reports can be automated for export to csv file**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should be able to automate the export of AskVA data reports to a csv file

- [x] **Scenario 17: Display number of currently open AskVA cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of currently open AskVA cases

- [x] **Scenario 18: Display number of currently closed AskVA cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of currently closed AskVA cases

- [x] **Scenario 19: Display number of open backlog AskVA cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of open backlog AskVA cases

- [x] **Scenario 20: Display number of closed backlog AskVA cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of closed backlog AskVA cases

- [x] **Scenario 21: Display number of routed AskVA cases**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of routed AskVA cases

- [x] **Scenario 22: Display number of cases routed from AskVA to T3 including EVSS**
  - Given that I am a DOMO user
  - When I access the dashboard
  - Then I should see a grid displaying the number of cases routed from AskVA to T3 including EVSS

**Comment by ATMiddleton** [comment]
2/7/24 - @Kimberley2023-CSM will contact "Jaime" at AskVA to confirm ability to export csv file from Microsoft Dynamics to csv files.  Also, will confirm automation of data migration to csv file and frequency of export.

**Comment by clantosswett** [comment]
@newworld2616 or @ATMiddleton could you upload the domo dashboard update presentation here so I can take a look? Thanks!

**Comment by ATMiddleton** [comment]
<img width="559" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM2xqQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--810c96f7092d96d66afa21029ffe23a8b2396b24/image.png" alt="image.png" />

**Comment by clantosswett** [comment]
@ATMiddleton @newworld2616 the presentation looks great. I have two questions, one of which might require a longer conversation.  

1. What data are we pulling for the closed backlog cases? 
2. Would it be possible to add another metric for how long it takes to close tier 2 and tier 3 cases? This might be a bigger question because we haven't done this in the past but for the new set of OKRs that were set by Charles, this is a baseline metric that OCTO would like to track to determine progress over the year. Maybe we can talk about this next week during our Medallia sync.

**Comment by ATMiddleton** [comment]
The team met with Mason Crane to review the draft proposed updated DOMO dashboard(s).  We determined the current limitations and major pain points.  We identified the metrics we want to report including - number of Open/Closed/Routed/Backlog cases for Tier II, Tier III, Tier III EVSS and Ava cases. 

Mason will present a final draft by 3/1/24

@clantosswett - the backlog information includes, date cases arrived and were closed as well as categorized by Tier II, Tier III, Tier III EVSS.
